### DISCUSSION TITLE
lighthouse report for mix of angular and react application

### DISCUSSION THREAD

**Discussion by swatigautam011** [GoogleChrome/lighthouse#16124] [Q&A] [answered]
I have a requirement for client side performance, where my application is a mix of Angular and React code modules. I need to do client side performance testing only specific to the React module, and because of that, I am using timespan mode so that I can only get React related diagnoses and the score, but in the report, I can see some CSS and JS for Angular. Is there any way we can get a lighthouse report only specific to react?

**Comment by adamraine** [comment, marked as answer]
No. Lighthouse observes everything that is on your page. If there is some Angular code still on the page then that will be taken into account for the Lighthouse report.
